src/apprt/gtk/Window.zig

```diff
@@ -629,6 +647,18 @@ fn gtkRefocusTerm(v: *c.GtkWindow, ud: ?*anyopaque) callconv(.C) bool {
     return true;
 }
 
+fn gtkRealize(v: *c.GtkWindow, ud: ?*anyopaque) callconv(.C) bool {
+    const self = userdataSelf(ud.?);
+
+    self.wayland = Wayland.init(v) catch return false;
+
+    if (self.wayland) |*wayland| {
+        wayland.setBlur(self.app.config.@"background-blur-radius" > 0) catch return false;
```

Comment:
We shouldn't duplicate this. In the macOS app it's common for there to be a central `syncAppearance` function that we call from everywhere where we need to setup appearance. We may want to copy that pattern here.